src/Dialogs/Settings/Panels/LayoutConfigPanel.cpp

```diff
@@ -186,6 +187,14 @@ LayoutConfigPanel::Prepare(ContainerWindow &parent, const PixelRect &rc)
           _("A list of possible InfoBox layouts. Do some trials to find the best for your screen size."),
           info_box_geometry_list, (unsigned)ui_settings.info_boxes.geometry);
 
+  AddInteger(_("InfoBox title size"),
+             _("An InfoBox has 3 lines: the value in the middle, "
+               "the title in the top line and the comment in the bottom line. "
+               "This parameter scales title and comment."),
```

Comment:
If it's called `scale_title_font`, then why does it scale the comment? This is confusing.

Reply:
Because the same font is used for title and comment: title_font